wGetStarted/MacroSecurity.md

```diff
@@ -0,0 +1,52 @@
+### TEST 16: Drill To An Altered Macro File
```

Comment:
is there a page with TESTS or am i missing something? i dont understand what TEST 16 means here.

Reply:
Sorry, I will delete this file, it is not supposed to be here.

Reply:
I deleted this file, it should not be here.